pkg/operation/common/utils.go

```diff
@@ -199,7 +199,7 @@ func deleteLokiPvc(ctx context.Context, k8sClient client.Client, namespace strin
 	defer cancel()
 	deadline, _ := ctx.Deadline()
 
-	if err := wait.PollUntilWithContext(ctx, 1*time.Second, func(context.Context) (done bool, err error) {
+	if err := wait.PollUntilContextCancel(ctx, 1*time.Second, true, func(context.Context) (done bool, err error) {
```

Comment:
We are passing `true` for immediate, this is the behaviour of `wait.PollImmediateWithContext` and not `wait.PollUntilWithContext`, Is this fine?